client/boot/index.js

```diff
@@ -87,7 +88,15 @@ function setUpContext( layout ) {
 		var parsed = url.parse( location.href, true );
 
 		context.layout = layout;
-		context.reduxStore = reduxStore;
+		//( function() {
+		//	var Dispatcher = require( 'dispatcher' );
+		//	Dispatcher.register( function( payload ) {
+		//		var action = payload.action;
+		//		console.log( 'Flux -> Redux' );
+		//		reduxStore.dispatch( action );
+		//	} );
+		//}() );
```

Comment:
> Interesting. Did this work? :)

It did. :)

> we probably don't need to integrate the two dispatchers now

I agree. I pushed it for the sake of communicating the exploration process, i.e. as a way of saying that this in particular had been tried.

> We should check we're not relying on any 'external' actions, although from what I can remember, we're not.

Yeah, IIRC only the reverse happens, and that's something we can handle at the Redux middleware level.